task/06-conditions-n-loops-tasks.js

```diff
@@ -270,7 +343,16 @@ function isCreditCardNumber(ccn) {
  *   165536 (1+6+5+5+3+6 = 26,  2+6 = 8) => 8
  */
 function getDigitalRoot(num) {
-    throw new Error('Not implemented');
+    let resultSum = num;
+    let newArr = [];
+    let reducer = (a,b) => parseInt(a) + parseInt(b);
+
+    while (resultSum > 9) 
+    {
+        newArr = resultSum.toString().split("")
+        resultSum = newArr.reduce(reducer)
```

Comment:
You can write it like:
resultSum = resultSum.toString().split("").reduce(reducer)
So newArr param is no needed anymore and your code is shorter :)

Reply:
done